package.json

```diff
@@ -33,7 +33,7 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.800.0",
-    "@angular-devkit/schematics": "7.1.3",
+    "@angular-devkit/schematics": "^8.0.3",
     "@angular/cli": "8.0.2",
```

Comment:
Thanks @suprishi. Be careful with the dependencies. We should align the angular dependencies, and in this case you should use the same for schematics and cli. Something like this:

```json
"@angular-devkit/schematics": "~8.0.3",
"@angular/cli": "~8.0.3",
```

As a rule we should use the same schema for Angular dependencies. Maybe just `~8.0.0` is enough for Angular core dependencies and for build, cli and others try to check the aligned versions.

Please change this accordingly to merge this PR. Thanks!

Reply:
Made the changes. Do have a look when time permits.